### PR TITLE
Exclude container-images path from current CI tests

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
We should add tests in this area, but for now, let's just make things more efficient (https://github.com/ehelms/foreman-quadlet/issues/23)

**Comment by evgeni** [comment]
Shouldn't this PR have triggered tests still?

**Comment by ehelms** [comment]
Should it?

**Comment by evgeni** [comment]
The PR changes files outside of container images, so yes

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -5,7 +5,11 @@ on:
   push:
     branches:
       - master
+    paths:
+      - '!container-images/**'
```
Comment (evgeni) [inline .github/workflows/test.yml:9]:
```suggestion
    paths-ignore:
      - 'container-images/**'
```
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

the bang-syntax is only to remove previously included paths, not to ignore whole structures

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -5,7 +5,11 @@ on:
   push:
     branches:
       - master
+    paths-ignore:
+      - 'container-images/**'
   pull_request:
+    paths:
+      - '!container-images/**'
```
Comment (evgeni) [inline .github/workflows/test.yml:12]:
```suggestion
    paths-ignore:
      - 'container-images/**'
```

**Review by evgeni — APPROVED** [review]

**Pull request merged**